zone/entity.cpp

```diff
@@ -2244,6 +2244,18 @@ Raid* EntityList::GetRaidByBot(const Bot* bot)
 	return nullptr;
 }
 
+Raid* EntityList::GetRaidByName(const char* name)
+{
+	for (const auto& r : raid_list) {
+		for (const auto& m : r->members) {
+			if (strcasecmp(m.member_name, name) == 0) {
```

Comment:
`if (Strings::Equals(m.member_name, name))`

Reply:
Good morning Kingly.  I don't seem to have the member Equals in the Strings class?

Reply:
Updated here and a few other places where this pattern was being used,